cmd/init.go

```diff
@@ -62,11 +62,12 @@ func (c *InitCommand) Run(args []string) int {
 	s.FinalMSG = "✓ Dependencies installed\n"
```

Comment:
🤦 I just noticed from your screenshot this outputs regardless of success or failure which isn't good. There might be an easy fix for that 🤔

Reply:
I think this might work:

```diff
        pip := exec.Command("pip", "install", "-r", "requirements.txt")

+       var finalMsg string
        s := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
        s.Suffix = " Installing pip dependencies (pip install -r requirements.txt) ..."
-       s.FinalMSG = "✓ Dependencies installed\n"
+       s.FinalMSG = finalMsg
        s.Start()

-       err = pip.Run()
+       pipCombinedOutput, err := pip.CombinedOutput()

        if err != nil {
+               finalMsg = "✗ Dependencies not installed\n"
                s.Stop()
-               c.UI.Error(fmt.Sprintf("Error installing pip requirements: %s", err))
+               c.UI.Error(fmt.Sprintf("Error installing pip requirements: \n%s", pipCombinedOutput))
                return 1
        }

+       finalMsg = "✓ Dependencies installed\n"
        s.Stop()

        return 0
```